src/client.rs

```diff
@@ -424,8 +422,8 @@ mod tests {
     fn demo_scan_status() {
         let cli = test_util::demo_site().unwrap();
         let (status, n) = cli.scan_status().unwrap();
-        assert_eq!(status, false);
-        assert_eq!(n, 521);
```

Comment:
Ideally there'd probably be a way that we don't depend on a server that isn't under the project's control for this, but oh well. It's pretty easy to spot when and why this fails.

Reply:
Yeah, this isn't ideal. We could loosen the assertion here if you'd like, something like `n !=0` perhaps?